src/Fmt.ml

```diff
@@ -102,29 +102,17 @@ let pre_break n s o fs = Format.pp_print_pre_break fs n s o
 
 (** Conditional on breaking of enclosing box ----------------------------*)
 
-let fits_breaks ?(force_fit_if = false) ?(force_break_if = false) fits
-    breaks fs =
-  let n, o, b =
-    let len = String.length breaks in
-    if len >= 2 && Char.equal breaks.[0] '@' then
-      let b = String.sub breaks ~pos:2 ~len:(len - 2) in
-      match breaks.[1] with
-      | ';' -> (
-        try Scanf.sscanf breaks "@;<%d %d>%s" (fun x y z -> (x, y, z))
-        with Scanf.Scan_failure _ | End_of_file -> (1, 0, b) )
-      | ',' -> (0, 0, b)
-      | ' ' -> (1, 0, b)
-      | _ -> (0, Int.min_value, breaks)
-    else (0, Int.min_value, breaks)
-  in
+let fits_breaks ?(force_fit_if = false) ?(force_break_if = false)
+    ?(hint = (0, Int.to_int Int.min_value)) fits breaks fs =
```

Comment:
`Int.to_int` ? This is definitively not needed.

Reply:
Done.